snipssonos/adapters/tts_sentence_adapter.py

```diff
@@ -0,0 +1,31 @@
+from snipssonos.shared import feedback
+from snipssonos.exceptions import ExternalDeviceDiscoveryUnreachable
+
+
+class TTSSentenceGenerator(object):
+    SUPPORTED_LANGUAGES = ["FRENCH", "ENGLISH"]
+
+    def __init__(self, lang):
+        self.language = lang
+
+    @property
+    def language(self):
+        return self._language
+
+    @language.setter
+    def language(self, lang):
+        if not (lang in self.SUPPORTED_LANGUAGES):
+            class_name = self.__class__.__name__
+            raise AttributeError(
+                "Tried to assign an unsupported language to the language property of {}".format(class_name))
+        else:
```

Comment:
Please, get rid of the else you do not need it. If the previous condition passes then this part of the code is never reached anyways.

Reply:
I meant this: 
~~~python
if not (lang in self.SUPPORTED_LANGUAGES):
       class_name = self.__class__.__name__
       raise AttributeError(
       "Tried to assign an unsupported language to the language property of {}".format(class_name))

self._language = lang
~~~